src/vaadin-number-field.html

```diff
@@ -31,21 +39,23 @@
       }
 
       [part="value"] {
-        -moz-appearance: textfield;
+        -moz-appearance: textfield !important;
```

Comment:
Why `!important`?

Reply:
Added comment. Without it visual tests fail with Firefox.